src/drivers/PinMap.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+namespace Pinetime {
+  namespace PinMap {
+    
+    #ifdef WATCH_P8
+      // COLMI P8
+      static constexpr uint8_t Charging = 19;
+      static constexpr uint8_t Cst816sReset = 13;
+      static constexpr uint8_t Button = 17;
+    #else
+      // Pinetime
+      static constexpr uint8_t Charging = 12;
+      static constexpr uint8_t Cst816sReset = 10;
+      static constexpr uint8_t Button = 13;      
+    #endif
+
+    static constexpr uint8_t Cst816sIrq = 28;
+    static constexpr uint8_t PowerPresent = 19;
+
+    static constexpr uint8_t Motor = 16;
+
+    static constexpr uint8_t LcdBacklight1 = 14;
+    static constexpr uint8_t LcdBacklight2 = 22;
+    static constexpr uint8_t LcdBacklight3 = 23;
```

Comment:
Maybe these pins should be renamed to BacklightLow, BacklightMedium and BacklightHigh.

Reply:
The naming is consistent with schematics, I would keep that.
https://files.pine64.org/doc/PineTime/PineTime%20Schematic-V1.0a-20191103.pdf

However this opens another conversation as I'm looking at this brightness schematics part first time. Lower brightness actually doesn't save power - it burns the rest energy in the resistor. So it might be optimized with PWM and we will use only Light1 wit 30 ohm resistor which burns least power. I might investigate it later in different PR.
![image](https://user-images.githubusercontent.com/497212/128170458-555203ba-3ff5-4bdc-b831-f0b90dd71a74.png)

Reply:
[sorry I had to delete my previous answer]
The naming is really not consistent with schematics, I would keep 1 2 3, but swap them, so pin 14 is backlight3
https://files.pine64.org/doc/PineTime/PineTime%20Schematic-V1.0a-20191103.pdf

However this opens another conversation as I'm looking at this brightness schematics part first time. Lower brightness actually doesn't save power - it burns the rest energy in the resistor. So it might be optimized with PWM and we will use only Light1 wit 30 ohm resistor which burns least power. I might investigate it later in different PR.

![image](https://user-images.githubusercontent.com/497212/128171082-8cb1afa2-0f10-4e47-b4d9-13dbf391f8d8.png)

Reply:
>Lower brightness actually doesn't save power - it burns the rest energy in the resistor.

That's not quite how resistors work. Higher resistance lowers current, which lowers power consumption.

>Even without PWM there might be more levels using variants of those 3 signals.

I noticed this too and tested this some time ago. There is one brightness setting that could be achieved between high and medium, the rest aren't noticable. Still this level is just slightly dimmer than max, and if we included this, the step from low to mid would be much larger than the other steps.

>This PWM would allow gradient change of brightness. Not sure there is a reason why the hardware does not use PWM.. because of EMI, not fast/efficient FET transistor?

PWM could be used to smooth transitions and achieve an even dimmer minimum brightness. I don't know if it would be useful to replace the existing steps with it.

>The naming is really not consistent with schematics, I would keep 1 2 3, but swap them, so pin 14 is backlight3

Having actual words describing the pins would still be clearer than numbers, since different pins affect the brightness different amounts.